models/classes/pack/ItemPack.php

```diff
@@ -41,13 +41,23 @@
  */
 class ItemPack implements JsonSerializable
 {
-
     /**
      * The supported assets types
      * @var string[]
      */
-    private static $assetTypes = ['html', 'document', 'js', 'css', 'font', 'img', 'audio', 'video', 'xinclude', 'apip'];
-
+    private static $assetTypes = [
+        'html',
+        'document',
+        'js',
+        'css',
+        'font',
+        'img',
+        'audio',
+        'video',
+        'xinclude',
+        'apip',
+        'pdf'
+    ];
```

Comment:
Maybe a constant can be used here?
These values are static and do not change anywhere.